CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Prism Themes Changelog
 
+## 1.9.1 (????-??-??)
+
+### New Themes
+
+* Tokyo Night Dark([#148](https://github.com/PrismJS/prism-themes/pull/148))
```

Comment:
I appreciate the sentiment, but the Prism team will update the changelog before the new release. No need to do that here.

Reply:
Sounds good, some repos wants you to add the changes.